js/project_details.js

```diff
@@ -0,0 +1,50 @@
+},
+{
+  LinkToSource: 'https://github.com/souad988/e-com',
+  linkToLiveVersion: 'https://souad988.github.io/e-com/',
+  image: './media/images/projects/youtube.png',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  name: 'ecom',
+  technologies: ['Html', 'css', 'js', 'python', 'django'],
+},
+{
+  LinkToSource: 'https://github.com/souad988/youtube',
+  linkToLiveVersion: 'https://souad988.github.io/youtube/',
+  image: './media/images/projects/youtube.png',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  name: 'youtube1',
+  technologies: ['Html', 'css'],
+},
+{
+  LinkToSource: 'https://github.com/souad988/portfolio',
+  linkToLiveVersion: 'https://souad988.github.io/portfolio/',
+  image: './media/images/projects/youtube.png',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  name: 'portfolio1',
+  technologies: ['Html', 'css', 'javascript'],
+},
+{
+  LinkToSource: 'https://github.com/souad988/e-com',
+  linkToLiveVersion: 'https://souad988.github.io/e-com/',
+  image: './media/images/projects/youtube.png',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  name: 'ecom1',
+  technologies: ['Html', 'css', 'js', 'python', 'django'],
+}];
+
+export default projectsDetail;
```

Comment:
- [ ] I'd suggest you follow the Figma design to improve the UI(user interface) of your page and kindly fix the broken image link too😎👍
- Your popup design 
![screenshot (4)](https://user-images.githubusercontent.com/44624138/151263702-69db6dd5-48ad-4c50-9fc2-8b6d7288a41f.png) | ![screenshot (5)](https://user-images.githubusercontent.com/44624138/151263731-3288c0e1-ee01-4d30-b6aa-9b3f8e5598a3.png)

- Figma design

![screenshot (6)](https://user-images.githubusercontent.com/44624138/151263870-f7d3e06b-2a16-4268-814e-a269f6faa10b.png) | ![screenshot (7)](https://user-images.githubusercontent.com/44624138/151263976-89f83788-57e8-458f-9c3c-aa0f506597d5.png)

Reply:
Hi @rotimiazeez 
thanks for reviewing, but for the imgs i don't know why they don't show up on github deployment while they do on my local machine if you can help with this i appreciate